src/pages/projectSettings/tabs/GeneralTab/getFormSchema.tsx

```diff
@@ -45,6 +45,7 @@ export const getFormSchema = (
           repositoryInfo: {
             type: "object" as "object",
             title: "Repository Info",
+            required: ["owner", "repo"],
```

Comment:
Let's make branch required too since [it is on legacy](https://github.com/evergreen-ci/evergreen/blob/a4c509ab4daeddd20465bcebf57aebe6a355a7f4/service/project.go#L327-L330)

Reply:
done!